recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,8 @@
-MACOSX_SDK_VERSION:        # [osx and x86_64]
-  - 10.12                  # [osx and x86_64]
+MACOSX_SDK_VERSION:  # [osx and x86_64]
```

Comment:
Can you add a comment in the description why this is necessary?

Reply:
clock_gettime missing in old osx releases

Reply:
can you maybe add this as a comment in the code or even just in the PR description? it always helps, also in the future when this gets updated.